packages/cli-tools/src/fetch.ts

```diff
@@ -36,7 +36,9 @@ const fetchToTemp = (url: string): Promise<string> => {
         }
 
         const dest = fs.createWriteStream(tmpDir);
-        const body = stream.Readable.fromWeb(result.body);
+        const body: stream.Readable = stream.Readable.fromWeb(
+          result.body as any,
+        );
```

Comment:
with the fix in npm.ts, I think we can remove this, no?

Reply:
Will remove if this is the case.

Reply:
Done.